pkg/exporter/process_test.go

```diff
@@ -63,10 +67,12 @@ func TestExportingProcess_SendingTemplateRecordToLocalTCPServer(t *testing.T) {
 	// Add template to exporting process
 	reg := registry.NewIanaRegistry()
 	reg.LoadRegistry()
+	antreaReg := registry.NewAntreaRegistry()
+	antreaReg.LoadRegistry()
```

Comment:
You seem to be using the global registry. Do you still need this?

Reply:
Will make the change in global registry PR